tests/ingest_runs/test_genomic_data_loader.py

```diff
@@ -0,0 +1,396 @@
+from creator.ingest_runs.genomic_data_loader import GenomicDataLoader
+from creator.ingest_runs.genomic_data_loader import DATASERVICE_URL
+from tests.integration.fixtures import test_study_generator # noqa F401
+
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+from kf_lib_data_ingest.etl.load.load import LoadStage
+
+import ast
+import os
+import pandas as pd
+import pytest
+
+
+@pytest.fixture()
+def study_id():
+    return "SD_YE0WYE0W"
+
+
+@pytest.fixture()
```

Comment:
Should probably be a module scoped fixture so that we're not running ingest_study every time this fixture is used in a test

Reply:
We can't do this currently as-is because it relies on the `tmpdir` fixture which is scoped for functions